src/java.base/share/conf/security/java.security

```diff
@@ -25,6 +25,8 @@
 #
 # If this properties file fails to load, the JDK implementation will throw
 # an unspecified error when initializing the java.security.Security class.
+# Properties in this file are typically parsed only once. Any modification
+# would require an application restart to reflect changes.
```

Comment:
Maybe change the second sentence to "If any of the properties are modified, applications should be restarted to ensure the changes are properly reflected."

This makes it more consistent with the words in the first sentence.

Reply:
Thanks for the review. Change implemented.